### DISCUSSION TITLE
Any guides to installing new models?

### DISCUSSION THREAD

**Discussion by michaelsft** [Anjok07/ultimatevocalremovergui#1351] [Q&A]
Hi everyone, I'm no programmer but I am fairly intuitive and I wanted to try the bandit cinematic separation model that someone linked to (to extract an audio commentary from a film soundtrack). Unfortunately for me, I have absolutely no idea how to load this into UVR5. Are there any guides or youtube videos available with instructions for a newbie like me?

Thanks for reading
Michael

**Comment by Gravitationalwavesarecool** [comment]
Hello! I recently learned from this YouTube video below, he covers a lot of information you'll need to begin! 

https://youtu.be/ykOKwz3eRUQ?si=Re64i_lwJsLcY2l-

Further, to load in a project easily open your file outside of the program and you can drag and drop into the input, and you can do the same with the folder location you want for the output!

**Reply by austinpharper** [reply]
This video does not address how to add additional models.

**Reply by Gravitationalwavesarecool** [reply]
Load to the 8 minute time stamp, it does

**Comment by hronik1** [comment]
I was also struggling. Here's a timestamp to showing how https://www.youtube.com/watch?v=p4YSfh-FbAs&t=260s. TL;DW: click wrench next to "Start Processing" Button -> "Download Center" -> Click radio button next to desired model and select models in the drop down.

**Reply by austinpharper** [reply]
That is not what he is asking. The question is how to install other models like Bandit that are not in the dropdowns.
